python/ray/scripts/scripts.py

```diff
@@ -524,6 +524,13 @@ def debug(address):
     default=False,
     help="If True, the usage stats collection will be disabled.",
 )
+@click.option(
+    "--labels",
```

Comment:
maybe `hidden=True,` for now and we can make it public after we finish everything